lib/content-services/content-node-selector/content-node-selector-panel.component.spec.ts

```diff
@@ -21,7 +21,6 @@ import { By } from '@angular/platform-browser';
 import { MinimalNodeEntryEntity, SiteEntry, SitePaging } from 'alfresco-js-api';
```

Comment:
I guess in those tests is missing the async

Reply:
Nice catch, I added the missing async. Please check now.

Reply:
I've added more async to the tests. Please check if it's all good now. Thank you.